APOLLO_SERVER2/src/typeDefs.js

```diff
@@ -18,15 +38,34 @@ const typeDefs =  gql`
     Percentage: String
     Second: String
     PathImage: String
+    name: String!
+    filePath: String!
+  }
+
+  type filesMetadata
+  {
```

Comment:
The keys {} must follow the same format as the previous ones, such as: 
type filesMetadata {
    name: String
    filePath: String
}